docs/public/content/docs/index.md

```diff
@@ -100,7 +100,7 @@ render(<App />, document.body);
 
 ```diff
 -import { render } from 'preact';
-+import hydrate from 'preact-iso/hydrate';
```

Comment:
Don't be fooled by this diff of a diff :D

Reply:
LOL wow that really throws GitHub for a loop